packages/react-magma-dom/src/components/StyledButton/styles.ts

```diff
@@ -104,25 +128,33 @@ export function buildButtonBackground(props) {
 export function buildBorderColor(props) {
   props = buildPropsWithDefaultButtonStyles(props);
 
-  if (props.disabled && props.isInverse && props.variant === 'outline') {
-    return props.theme.colors.disabledInverseText;
-  }
   if (props.disabled) {
-    return props.theme.colors.neutral06;
+    if (props.isInverse) {
+      if (props.color === 'secondary') {
+        return transparentize(0.8, props.theme.colors.neutral100);
+      }
+      return 'none';
+    }
+    if (props.color === 'secondary') {
+      return props.theme.colors.neutral300;
+    }
+    return props.theme.colors.neutral300;
   }
+
   if (props.color === 'marketing') {
-    return props.theme.colors.pop04;
-  }
-  if (props.isInverse) {
-    return props.theme.colors.neutral08;
+    return props.theme.colors.secondary500;
   }
   if (props.color === 'secondary') {
-    return props.theme.colors.neutral05;
+    return props.theme.colors.primary300;
+  }
+  if (props.isInverse) {
+    if (props.color === 'secondary') {
+      return transparentize(0.5, props.theme.colors.tertiary500);
+    }
+    return props.theme.colors.neutral100;
   }
```

Comment:
Let's move this isInverse check to line 143 so if it's Inverse and Secondary, it returns the correct color